hq/app/api/Snyk.scala

```diff
@@ -80,18 +76,4 @@ object Snyk {
     } yield results
   }
 
-  def getSnykToken(configraun: Configuration): Attempt[SnykToken] = configraun.getAsString("/snyk/token") match {
```

Comment:
The app will now fail to start if these config values are not present. This makes it more consistent with other configuration e.g. https://github.com/guardian/security-hq/blob/035c0f8de5d1a7a50b2b065d5d6878f1b779e09f/hq/app/AppComponents.scala#L101